test/f90_correct/inc/remove_deep_deallocatable.mk

```diff
@@ -0,0 +1,25 @@
+#
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+
+########## Make rule for test remove_deep_deallocatable   ########
+
+
+remove_deep_deallocatable: run
```

Comment:
```
$(TEST): run
```

Reply:
Occurrences of `remove_deep_deallocate` in this file should be replaced with `$(TEST)`.